substrate/frame/treasury/src/lib.rs

```diff
@@ -499,7 +515,9 @@ pub mod pallet {
 			.unwrap_or(Ok(()))?;
 
 			let beneficiary = T::Lookup::lookup(beneficiary)?;
+			#[allow(deprecated)]
```

Comment:
are these needed when the whole function is marked as allow(deprecated)?

Reply:
I've added these everywhere where warnings were thrown by rust compiler

Reply:
We can remove #[deprecated] attractive after putting them into metadata, because as of right now they are not removed, only added to metadata